Common/Extensions.cs

```diff
@@ -3029,6 +3029,24 @@ public static string RemoveFromEnd(this string s, string ending)
             }
         }
 
+        /// <summary>
+        /// Returns a new string in which specified start in the current instance is removed.
+        /// </summary>
+        /// <param name="s">original string value</param>
+        /// <param name="start">the string to be removed</param>
+        /// <returns>Substring with start removed</returns>
+        public static string RemoveFromStart(this string s, string start)
+        {
+            if (!s.IsNullOrEmpty() && !start.IsNullOrEmpty() && s.StartsWith(start, StringComparison.InvariantCulture))
```

Comment:
nit - should use `string.IsNullOrEmpty` else it's  converting the string to an enumerable and enumerating it 😬